grakn-core/src/main/java/ai/grakn/util/REST.java

```diff
@@ -119,6 +119,7 @@ public static String resolveTemplate(String pathTemplate, String... pathParams)
             public static final String QUERY = "query";
             public static final String INFER = "infer";
             public static final String MULTI = "multi";
+            public static final String TYPE = "type";
```

Comment:
ooooo I really don't like type as the name of the parameter. Type is so overloaded in our world. Can we not use `tx-type` or `tx-mode` ?

Reply:
`TxType` ?